maquetado/scss/styles.scss

```diff
@@ -1,3 +1,12 @@
+.body-detail {
+  height: 100vh;
+  padding: 40px 60px;
+}
+
+.body-background {
+  background-color: $wild-sand;
```

Comment:
you're not importing the colors in this file

Reply:
@bhflm is already importing `colors.scss` in `application.scss` so it's no needed to import `colors.scss` here